extensions/trace_propagators/src/main/java/io/opentelemetry/extensions/trace/propagation/B3PropagatorExtractor.java

```diff
@@ -7,10 +7,12 @@
 
 import io.grpc.Context;
 import io.opentelemetry.context.propagation.TextMapPropagator;
+import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
 
 @Immutable
 interface B3PropagatorExtractor {
 
+  @Nullable
```

Comment:
Any thoughts about using `Optional<Context>` as the return type, now that we have java 8?

Reply:
Nice! I'm glad Java 8 is supported now :)
Yeh returning an ```Optional``` would be better than null.  I'll update it.